Text/CustomEntitySearch/README.md

```diff
@@ -21,7 +21,14 @@ These skills have no additional requirements than the ones described in [the roo
 
 ## Settings
 
-This function requires Latin-based text (as seen in the sample document provided).
+This function requires Latin-based text (as seen in the sample document provided). The input field "words" is optional, where a user can add a "words.json" file instead.
+
+## Sample Config File
+```json
+{
+	["foo1", "foo2"]
```

Comment:
You forgot the "targetWords" property name. Then again, see my comment below on simplifying the format of the file.